index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <h3>The man who saved a billion lives</h3>
+      <div id="img-div">
+        <img  id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+        <br>
+        <figcaption id="img-caption">An image of Dr. Norman Borlaug</figcaption>
+      </div>
+      <h2 id="tribute-info">Here's a time line of Dr. Borlaug's life:</h2>
+
+      <strong> 
```

Comment:
Please use the section tag and do your style in the CSS file to make the code more readable.